recipes/jill/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "jill" %}
+{% set version = "0.11.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jill-{{ version }}.tar.gz
+    sha256: 17328a3769f3975018257e98d1b23c6caaed07a73c6c35947df4aebdef206b3f
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - jill=jill.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - fire
+    - jsonschema
+    - python-gnupg
+    - requests
+    - requests-futures
+    - semantic_version
+    - wget
```

This is depending on the GNU `wget` command line utility, not the [PyPI](https://pypi.org/project/wget/) package, which has not yet been packaged for conda-forge.

---

@johnnychen94 Can you instead use the PyPI `wget` or `requests` as you were suggesting? Perhaps that will help circumvent this hurdle. 

@BastianZim @dopplershift Thank you, for looking into it.

---

@sugatoray To clarify: `jill` already depends on the PyPI `wget` package. The problem is that as written above, the recipe is requesting conda-forge's `wget` package, which downloads the commandline `wget` package, which is why `import wget` fails. I thought the PyPI version wasn't available, but I was wrong. This line just needs to change to:
```suggestion
    - python-wget
```

---

@sugatoray Just as a general FYI, Grayskull maintains a mapping of PyPI to CF names here: https://github.com/conda-incubator/grayskull/blob/main/grayskull/strategy/config.yaml

If you find these kinds of exceptions please feel free to add them there.

---

@BastianZim I got this pypi-cf name-mapping from `@beckermr` (as far as I recall).

- https://github.com/regro/cf-graph-countyfair/blob/master/mappings/pypi/grayskull_pypi_mapping.yaml